README.md

```diff
@@ -269,11 +362,15 @@ While we each bring a different perspective and set of expertise to customer int
 
 # Company policies: our expectations of each other
 
+## Information security and privacy :lock:
+
+Able Health's customers entrust us with a significant amount of protected health information (PHI). As such, we take great care to ensure the security and privacy of potentially sensitive information. All team members receive HIPAA training and may refer to our HIPAA policies and procedures at any time for further details.
+
 ## Team meetings :clipboard:
 
-* Product standup at 9:50am every day
+* Product standup at 9:45am every day
```

Comment:
Ah ok. We've always called that the product stand-up since it should include PMs and designers as well, i.e. anyone who operates within the product sprint schedule. Any issue with that? I will move it to 9:30 either way.